tailwind.config.ts

```diff
@@ -8,11 +8,15 @@ const config: Config = {
   ],
   theme: {
     extend: {
-      backgroundImage: {
-        'gradient-radial': 'radial-gradient(var(--tw-gradient-stops))',
-        'gradient-conic':
-          'conic-gradient(from 180deg at 50% 50%, var(--tw-gradient-stops))',
-      },
+      colors: {
+        // Background Colors
+        // TODO: Develop a better naming scheme - dont name after color because theme change
```

Comment:
Lets get @Prabal-Kochhar to develop a fully fledged color scheme for the site that way we have good color names and such from the get-go

Reply:
I agree I think I feel comfortable merging with the way it is now and changing things arround once that's done though